extensions/amp-analytics/0.1/vendors.js

```diff
@@ -59,6 +64,17 @@ export const ANALYTICS_CONFIG = {
           'st=${socialActionTarget}'
     },
     'optout': '_gaUserPrefs.ioo'
+  },
+
+  'parsely': {
+    'host': 'srv.pixel.parsely.com',
```

Comment:
The config spec has changed. The hostname is now part of the request. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#requests